lib/Raven/Processor/SanitizeHttpHeadersProcessor.php

```diff
@@ -67,6 +67,6 @@ public function process(&$data)
      */
     private function getDefaultHeaders()
     {
-        return ['Authorization', 'Proxy-Authorization', 'X-Csrf-Token', 'X-CSRFToken', 'X-XSRF-TOKEN'];
+        return ['Authorization', 'Proxy-Authorization', 'X-Authorization', 'Cookie', 'X-Csrf-Token', 'X-CSRFToken', 'X-XSRF-TOKEN'];
```

Comment:
As mentioned by @ste93cry, if you remove the `Cookie` header this PR can be accepted.